data/rao-result/rao-result-api/src/main/java/com/farao_community/farao/data/rao_result_api/RaoResultClone.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.rao_result_api;
+
+import com.farao_community.farao.commons.Unit;
+import com.farao_community.farao.data.crac_api.Instant;
+import com.farao_community.farao.data.crac_api.RemedialAction;
+import com.farao_community.farao.data.crac_api.State;
+import com.farao_community.farao.data.crac_api.cnec.FlowCnec;
+import com.farao_community.farao.data.crac_api.cnec.Side;
+import com.farao_community.farao.data.crac_api.network_action.NetworkAction;
+import com.farao_community.farao.data.crac_api.range_action.PstRangeAction;
+import com.farao_community.farao.data.crac_api.range_action.RangeAction;
+
+import java.util.*;
+
+/**
+ * clone that reproduces the results obtained by RaoResultImpl in order to override some methods without rewriting all the implementations
```

Comment:
```suggestion
 * Clone that reproduces the results obtained by another RaoResult instance. It can be used to override some methods without rewriting all the implementations
```